app/domain/readability/entities/filing_readability.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module SECond
+  module Entity
+    # Entity for file contributions
+    class FilingReadability
+      include Mixins::ReadabilityCalculator
+
+      # WANTED_LANGUAGES = [
```

Comment:
same problem with commented-out code. you can use [`wiki`](https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages)